### PR TITLE
Singularity options at the Crick

### PR DESCRIPTION
**Pull request by ChristopherBarrington**
---
name: Modification to existing Crick config
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [ ] Add your custom config file to the `conf/` directory
- [ ] Add your documentation file to the `docs/` directory
- [ ] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [ ] Add your custom profile to the `README.md` file in the top-level directory
- [ ] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

---

Some paths may not be accessible when using Singularity. Adding `/nemo` and `/flask` explicitly to `runOptions` allows paths to files to be correctly resolved. `autoMounts` is already enabled so I'm not sure why these paths are not exposed.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -132,6 +132,7 @@ jobs:
           - "wehi"
           - "wustl_htcf"
           - "xanadu"
+          - "ChristopherBarrington"
```
Comment (maxulysse) [inline .github/workflows/main.yml:135]: Are you a profile @ChristopherBarrington ?

Comment (maxulysse) [reply .github/workflows/main.yml:135]:
```suggestion
```

Comment (ChristopherBarrington) [reply .github/workflows/main.yml:135]: 🤣 I shouldn't be! It should only have been there for the testing part though, right?

Comment (maxulysse) [reply .github/workflows/main.yml]: I don't think you even need that for testing

**Review by maxulysse — APPROVED** [review]
LGTM

File: .github/workflows/main.yml (hunk 1 above)
Comment (ChristopherBarrington) [reply .github/workflows/main.yml]: > In order to ensure that the config file is tested automatically with GitHub Actions please add your profile name to the profile: scope

Of course that isn't MY profile name 🤦 I thought it was weird

**Comment by chris-cheshire** [comment]
Hi Chris, could you describe what this fixes, or enhances?

**Comment by ChristopherBarrington** [comment]
We have had problems with singularity not being able to access certain files when using the scratch space (/flask). This wasn't a problem before because scratch was under /camp which was reliably mounted.

**Review by chris-cheshire — APPROVED** [review]

**Pull request merged**